exaudfclient/base/python/python3/python_ext_dataframe.cc

```diff
@@ -493,9 +556,9 @@ inline void getColumnArrays(PyObject *colArray, int numCols, int numRows,
             PyPtr asType (PyObject_GetAttrString(array.get(), "astype"));
             PyPtr keywordArgs(PyDict_New());
             PyDict_SetItemString(keywordArgs.get(), "copy", Py_False);
-            PyPtr funcArgs(Py_BuildValue("(s)", colTypes[c].first.c_str()));
+      std::string numpyDTypeStr = numpyCTypeToNumpyDTypeStrMap.at(colTypes[c].second);
```

Comment:
Also "at" throws an exception if the key does not exist in the map!

Reply:
Hm, it should not happen in production, this would be a programming error. I will think about it.

Reply:
I think, we don't need a new error code for this. This will be caught by another error code and as I said, it should not happen during production, means it only happens during development, there we have tools for debugging.